include/envoy/config/subscription.h

```diff
@@ -97,20 +97,22 @@ using SubscriptionPtr = std::unique_ptr<Subscription>;
 /**
  * Per subscription stats. @see stats_macros.h
  */
-#define ALL_SUBSCRIPTION_STATS(COUNTER, GAUGE)                                                     \
+#define ALL_SUBSCRIPTION_STATS(COUNTER, GAUGE, TEXT_READOUT)                                       \
   COUNTER(init_fetch_timeout)                                                                      \
   COUNTER(update_attempt)                                                                          \
   COUNTER(update_failure)                                                                          \
   COUNTER(update_rejected)                                                                         \
   COUNTER(update_success)                                                                          \
   GAUGE(update_time, NeverImport)                                                                  \
-  GAUGE(version, NeverImport)
+  GAUGE(version, NeverImport)                                                                      \
+  TEXT_READOUT(version_text)
```

Comment:
Can we document this in the same place we document the gauge as well as add a release note?

Reply:
Done.